salt/modules/linux_lvm.py

```diff
@@ -486,10 +486,21 @@ def lvresize(size, lvpath):
 
 
         salt '*' lvm.lvresize +12M /dev/mapper/vg1-test
+        salt '*' lvm.lvresize lvpath=/dev/mapper/vg1-test extents=+100%FREE
+
     '''
-    ret = {}
-    cmd = ['lvresize', '-L', six.text_type(size), lvpath]
-    cmd_ret = __salt__['cmd.run_all'](cmd, python_shell=False)
-    if cmd_ret['retcode'] != 0:
-        return {}
-    return ret
+    if size and extents:
+        return 'Error: Please specify only one of size or extents'
+
+    cmd = ['lvresize']
+
+    if size:
+        cmd.extend(['-L', '{0}'.format(size)])
+    elif extents:
+        cmd.extend(['-l', '{0}'.format(extents)])
+    else:
+        return 'Error: Either size or extents must be specified'
```

Comment:
Same question here.

Reply:
Needed to be defensive since I had to change _size_ to keyword argument.
But as mentioned, won't affect others who use lvextend('+12M', '/dev/mapper/vg1-test')